mozilla-release/browser/components/sessionstore/nsSessionStartup.js

```diff
@@ -322,33 +310,21 @@ SessionStartup.prototype = {
            this._sessionType == Ci.nsISessionStartup.RESUME_SESSION;
   },
 
+  _willRestoreCliqz() {
```

Comment:
we need automated that for this feature. it is impossible to tell if its going to work after just looking at the code.

@marta-cliqz @tamara-cliqz could take care of it